motionplan/planManager.go

```diff
@@ -214,8 +222,16 @@ func (pm *planManager) planAtomicWaypoints(
 		}
 
 		pathPlanner := planners[i]
+
+		var maps *rrtMaps
+		if seedPlan != nil {
+			maps, err = pm.planToRRTGoalMap(seedPlan, goal)
+			if err != nil {
+				return nil, err
+			}
+		}
 		// Plan the single waypoint, and accumulate objects which will be used to constrauct the plan after all planning has finished
-		newseed, future, err := pm.planSingleAtomicWaypoint(ctx, goal, seed, pathPlanner, nil)
+		newseed, future, err := pm.planSingleAtomicWaypoint(ctx, goal, seed, pathPlanner, maps)
```

Comment:
In the case where we have a seedPlan doesnt this mean we are planning twice?

Reply:
`planToRRTGoalMap` just converts a `Plan` to a `*rrtMaps`, it doesn't plan anything.